Automattic/jetpack
# Enable Jetpack Forms package by default

## Proposed changes:

Once #28752 is merged, we'll have migrated all contact form related code to the new automattic/jetpack-forms package. In order to not have to maintain multiple codebases, we'd like to make the package the new default with the existing module becoming deprecated and eventually removed - possibly in the following jetpack version once we've confirmed everything is running smoothly.

### Other information:


## Jetpack product discussion



pcsBup-N3-p2

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






This PR doesn't contain any user facing changes, that's said it's good to check some of the following:

- Contact form blocks should still load as expected in the editor.
- Submitting feedback or contact form responses should still work correctly.
- You should be able to access the submitted responses in WP Admin under `Feedback`.
